recipes/myproxyclient/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MyProxyClient" %}
+{% set version = "1.4.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "5b7ad3cc3c3cd71595a9b0278a51c138699e229d47a41f2e2add72fbc12f58cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - myproxyclient = myproxy.script:main
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyopenssl
+    - certifi
+
+  run:
+    - python
+    - pyopenssl
```

Comment:
I'm not really sure. I added it because previous build failed with `setup.py cannot download certifi. Please add it to requirements in meta.yaml`.
We only need https://github.com/apache/climate/tree/master/ocw on conda-forge but as `myproxyclient` and `esgf-pyclient` are required for `climate(ocw)` I'm adding them to conda-forge. I took the recipes from here https://github.com/apache/climate/tree/master/conda_recipes and tweaked them. 

Is there any other way to get `climate` on conda-forge without uploading all these dependencies to conda-forge? (They are available on some other channels)